joss_paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Zarrtraj: A Python package for streaming molecular dynamics trajectories from cloud services'
+tags:
+  - streaming
+  - molecular-dynamics
+  - file-format
+  - mdanalysis
+  - zarr
+authors:
```

Comment:
You'll (1) need to have each author agree on authorship (see JOSS for what that means for them) and (2) need to come up with an order that is acceptable to everyone.

Reply:
@ljwoods2 normal protocol is to write an email to each author asking them if they would like to be an author if they haven't already agreed.

Reply:
You can CC myself and Oliver

Reply:
And see https://joss.readthedocs.io/en/latest/submitting.html#authorship — as an author you have responsibilities

> The authors themselves assume responsibility for deciding who should be credited with co-authorship, and co-authors must always agree to be listed. In addition, co-authors agree to be accountable for all aspects of the work, and to notify JOSS if any retraction or correction of mistakes are needed after publication.

I would also add that authors need to have read the manuscript and approve it (approval review on GitHub) and be willing to contribute to the writing if asked.